interface/DataTypes.h

```diff
@@ -16,7 +16,10 @@ enum value {
 	ElectronHad,
 	// MuHad,
 	SingleElectron,
-	SingleMuon,
+	SingleMuon_Run2016B,
```

Comment:
BTW: Do we really want to do it this way or do we want 1 dataset for data?
`SingeMuon_*` will match to `SingleMuon`. I do not see where in the code a RunX distinction is used.